tests/test_grid3d/grid_generator.py

```diff
@@ -0,0 +1,23 @@
+import hypothesis.strategies as st
+
+from xtgeo.grid3d import Grid
```

Comment:
Shall the file name be `test_grid_generator.py`? I was thinking it is practice that all modules for testing shall have a  `test`prefix(?)

Reply:
prefixing it with `test_` will make pytest discover it and run its tests. In this case we just want to be able to import it to other tests.